test/includes.spec.js

```diff
@@ -13,30 +13,36 @@ describe("includes", function () {
     function setupLinks(_ids) {
         ids = _ids;
 
-        function link(url, path, value) {
+        function link(resource, resourceId, link, linkId) {
+          const url = `/${resource}/${resourceId}`;
+
             return new Promise(function (resolve) {
-                var data = [
-                    {
-                        op: 'replace',
-                        path: path,
-                        value: value
+              var data = {
```

Comment:
Didn't change the test behaviour, it's passing, I just changed it to be an object as the PUT method expects.